webapp/channels/src/components/modal_tabs/index.tsx

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Was able to add navigation and selection through `enter`.

Tried adding for space bar too, but it space bar events weren't even making it to these tabs, even if I used capturing phase. I'm guessing it's because its in the modal. Not planning on adding space bar since I think it would require a lot more investigation of the modal and how/if we can circumvent whatever it is doing.